### DISCUSSION TITLE
Make -j showing error

### DISCUSSION THREAD

**Discussion by T-449** [open-quantum-safe/openssl#469] [Q&A]
Hi,
I am trying to build the project based on the steps mentioned in the README. Despite following it step by step the build fails. I am giving a screenshot of the failure.
![Screenshot from 2024-07-16 11-36-20](https://github.com/user-attachments/assets/2e627db1-abde-4596-9ae9-c8e3b7b47cc9).

**Comment by dstebila** [comment]
This project is no longer supported. Unless you have a reason to use an old version of OpenSSL, we recommend you use OpenSSL 3 combined with [oqs-provider](https://github.com/open-quantum-safe/oqs-provider/).

The build failure may be related to running `make -j` (if memory serves, building OpenSSL is sometimes unreliable with `-j`).  Otherwise I'm not sure.
